Protobuf failing on Bazel CI (Windows)

Build @head is failing on Windows.

https://buildkite.com/bazel/protobuf/builds/3732#5e9ea0bb-db47-4277-91bb-6aa9de40e48d

```
Executing tests from //java/core:TextFormatTest
-----------------------------------------------------------------------------
JUnit4 Test Runner
.E
Time: 0.016
There was 1 failure:
1) initializationError(com.google.protobuf.TextFormatTest)
...
Caused by: java.lang.IllegalArgumentException: Couldn't read file: C:\b\vxgrmk6i\execroot\com_google_protobuf\bazel-out\x64_windows-fastbuild\bin\src\google\protobuf\testdata\text_format_unittest_data_oneof_implemented.txt
	at com.google.protobuf.TestUtil.readBytesFromFile(TestUtil.java:3798)
	at com.google.protobuf.TestUtil.readTextFromFile(TestUtil.java:3761)
	at com.google.protobuf.TextFormatTest.<clinit>(TextFormatTest.java:83)
	... 36 more
Caused by: java.io.FileNotFoundException: C:\b\vxgrmk6i\execroot\com_google_protobuf\bazel-out\x64_windows-fastbuild\bin\src\google\protobuf\testdata\text_format_unittest_data_oneof_implemented.txt (The system cannot find the path specified)
	at java.io.RandomAccessFile.open0(Native Method)
	at java.io.RandomAccessFile.open(RandomAccessFile.java:316)
	at java.io.RandomAccessFile.<init>(RandomAccessFile.java:243)
	at com.google.protobuf.TestUtil.readBytesFromFile(TestUtil.java:3791)
	... 38 more

FAILURES!!!
Tests run: 1,  Failures: 1
```

--- a/java/core/src/test/java/com/google/protobuf/TestUtil.java
+++ b/java/core/src/test/java/com/google/protobuf/TestUtil.java
@@ -3758,7 +3758,7 @@ public final class TestUtil {
 
   /** @param filePath The path relative to {@link #getTestDataDir}. */
   public static String readTextFromFile(String filePath) {
-    return readBytesFromFile(filePath).toStringUtf8();
+    return readBytesFromFile(filePath).toStringUtf8().replace(System.getProperty("line.separator"), "\n");
   }
 
   private static File getTestDataDir() {

